test/Microsoft.Extensions.Logging.EventSource.Test/EventSourceLoggerTest.cs

```diff
@@ -10,6 +10,7 @@
 using Newtonsoft.Json;
 using Xunit;
 using Microsoft.Extensions.DependencyInjection;
+#pragma warning disable 618
```

Comment:
Can you add a comment for each `#pragma warning disable` to say what it's disabling and why?

Reply:
(Elsewhere this is `CS0618 // Type or member is obsolete`